hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/PipelineEndpoint.java

```diff
@@ -105,9 +104,9 @@ public Response getPipelines() {
               .setDatanodes(datanodes)
               .setDuration(duration)
               .setStatus(pipeline.getPipelineState())
-              .setReplicationFactor(ReplicationConfig
-                  .getLegacyFactor(pipeline.getReplicationConfig()).getNumber())
-          .setReplicationType(pipeline.getType().toString());
+              .setReplicationFactor(
```

Comment:
Would it make more sense to change the parameters here to accept just "ReplicationConfig" rather than Factor and Type separately? Then internally it can call the methods on ReplicationConfig it requires?